src/AppBundle/Helper/UserManager.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace AppBundle\Helper;
+
+use AppBundle\Entity\User;
+use Doctrine\ORM\EntityManager;
+
+class UserManager
+{
+    protected $_templateEngine;
+    protected $em;
+
+    private $rootDir;
+
+    public function setRootDir($rootDir)
+    {
+        $this->rootDir = $rootDir;
+    }
+
+    public function __construct($_templateEngine, EntityManager $em)
```

Comment:
for future for find city and country by ip with help GeoLite. Like this $this->rootDir.'/data/GeoLite2-City.mmdb